labs/Variables/readme.md

```diff
@@ -72,19 +72,19 @@ If you want a UCA or your instructor to check your answer to a previous lab, jus
 
 #. At the end of your `Main` method:
 
-    #. Add three statements that change the values of the three variables,
-    #. Copy the `Console.WriteLine` statement that was previously given,
-    #. Compile and run the program.
+    #. Add three statements that change the values of the three variables.
+    #. Copy the `Console.WriteLine` statement that was previously given.
+    #. Compile and execute the program.
```

Comment:
Yes. I'll have to look for other instances of "run" to replace them with "execute". Consistency is important.